howso/direct/client.py

```diff
@@ -5241,6 +5265,23 @@ def set_auto_analyze_params(  # noqa: C901
         *,
         auto_analyze_limit_size: Optional[int] = None,
         analyze_growth_factor: Optional[float] = None,
+        action_features: Optional[Iterable[str]] = None,
+        context_features: Optional[Iterable[str]] = None,
+        k_folds: Optional[int] = None,
+        num_samples: Optional[int] = None,
+        dt_values: Optional[Iterable[float]] = None,
+        k_values: Optional[Iterable[int]] = None,
+        p_values: Optional[Iterable[float]] = None,
+        bypass_hyperparameter_analysis: Optional[bool] = None,
+        bypass_calculate_feature_residuals: Optional[bool] = None,
+        bypass_calculate_feature_weights: Optional[bool] = None,
+        targeted_model: Optional[Literal["omni_targeted", "single_targeted", "targetless"]] = None,
+        num_analysis_samples: Optional[int] = None,
+        analysis_sub_model_size: Optional[int] = None,
+        use_deviations: Optional[bool] = None,
+        inverse_residuals_as_weights: Optional[bool] = None,
+        use_case_weights: Optional[bool] = None,
+        weight_feature: Optional[str] = None,
```

Comment:
Adding comment for tracking; expanding use of an attribute_map to parameters. Current changes applied still need docstring edits.

Reply:
auto_analyze_expanded: e435c3d5ddcfec8f413e942d13ee30985f76df03
docstrings added: 96ba4b6